src/tests/SearchView.test.tsx

```diff
@@ -66,13 +66,14 @@ describe('Search View', () => {
   });
 
   it('should fetch and display recent results when repository is selected', async () => {
-    const spyOnFetch = jest.spyOn(global, 'fetch').mockImplementation(() =>
+    global.fetch = jest.fn(() =>
```

Comment:
(not for this patch)
I moved our mocking of fetch to the `fetch-mock-jest` util in https://github.com/firefox-devtools/profiler/pull/4020, in case you want to have a look at this.